docs/server/push_duty/tag-services.md

```diff
@@ -0,0 +1,80 @@
+# Tag Services
+
+## Manually tagging a service
+
+`addons-server` is automatically tagged when a release is published. See [release-docs](./release-docs.md) for more information.
+
+For now, `addons-frontend` is still manually tagged.
+To create and push a new tag follow these steps:
+
+Tags are of the format: ``YYYY.MM.DD``,
+
+```{note}
+The date is the date of the push, not the date of tagging.
+```
+
+## Checking out the default branch
+
+To create a new tag, you have to checkout a branch that will be used for the tag.
+This is almost always the default branch of the repository.
+
+You can find out the default branch by running
+
+```bash
+gh repo view --json defaultBranchRef --jq '.defaultBranchRef.name'
+```
+
+```{note}
+You need to have the GitHub CLI installed to run this command.
+```
+
+## Creating and pushing the tag
+
+Once you've checked out the right branch, make sure you have the latest changes by running:
+
+```bash
+git pull
+```
+
+```{warning}
+Make absolutely sure you are pulling from the correct remote.
+```
+
+Create the tag by simply running the command:
+
+```bash
+git tag <tag-date>
+```
+
+Push the tag to the remote repository:
+
+```bash
+git push <remote> <tag-date>
+```
+
+Here is a full example:
+
+```bash
+git checkout main
+git pull
+git tag 2015.09.10
+git push origin 2015.09.10
+```
```

Comment:
Fwiw, I always do

```bash
git fetch <remote>
git checkout <remote>/<default-branch>
git tag 2015.09.10
git push <remote> 2015.09.10
```

Reply:
I think the version I have is copied from the existing doc. would you change it?

Reply:
I'm fine with leaving it - I'm not familiar enough to the nuances of git to know if my way has some edge cases that makes it less robust (although I've always thought it _more_ robust because it doesn't rely on a local `master` branch being in a state that can accept the `git pull`)